app/code/Magento/Cms/Model/Wysiwyg/Images/GetInsertImageContent.php

```diff
@@ -60,4 +88,43 @@ public function execute(
 
         return $this->imagesHelper->getImageHtmlDeclaration($filename, $renderAsTag);
     }
+
+    /**
+     * Retrieve size of requested file
+     *
+     * @param string $path
+     * @return int
+     */
+    public function getImageSize(string $path): int
```

Comment:
Please move these public methods to the introduced controller (as private methods).